typescript-selenium-webdriver/tests/index.test.ts

```diff
@@ -54,23 +55,77 @@ describe('index.html', () => {
         await driver.wait(until.elementLocated(By.css('h1')));
     }, TEST_TIMEOUT_MS);
 
-    it('only contains known accessibility violations', async () => {
-        // Run an accessibility scan using axe-webdriverjs
-        const axeResults = await AxeBuilder(driver).analyze();
+    // This test case shows the most basic example: run a scan, fail the test if there are any failures.
+    // This is the way to go if you have no known/pre-existing violations you need to temporarily baseline.
+    it('has no accessibility violations in the h1 element', async () => {
+        const accessibilityScanResults = await AxeBuilder(driver)
```

Comment:
This object and it's usage is a little confusing: `AxeBuilder` -> make me think we're building and Axe instance... at the end of the chain I can see `.analyze()`... which, well... doesn't seem to build me an Axe. I guess it's ok to have a `AxeRunner` or and `A11YAnalyzer` class that happens to use fluent syntax and the builder pattern-like API to set it up though.

Reply:
Renamed to AxeWebdriverjs to improve clarify